exercises/sum-of-multiples/sum-of-multiples.spec.js

```diff
@@ -1,39 +1,39 @@
-var SumOfMultiples = require('./sum-of-multiples');
+var sumOfMultiples = require('./sum-of-multiples');
```

Comment:
I would prefer having the variable starting with capital `S`. ESLint is complaining because the variable is used as a regular function, but in fact `SumOfMultiples` is a constructor function.

JavaScript allows to use constructor functions in both ways `SumOfMultiples()` and `new SumOfMultiples()`.

In this case, as `SumOfMultiples` is a constructor function, in order to fix ESLint errors, I would write `new SumOfMultiples([3, 5])` in line 5 for example. Does it makes sense to you?

If `SumOfMultiples` were a regular function, it would make a lot of sense to start it in lowercase.